Makefile

```diff
@@ -204,6 +204,8 @@ release: $(GITHUB_RELEASE)
 vendor: $(GO)
 	$(GO) mod tidy -compat=$(GO_VERSION)
 	$(GO) mod vendor
+	pushd tools && $(GO) mod tidy -compat=$(GO_VERSION)
+	pushd tools && $(GO) mod vendor
```

Comment:
there was no popd, so pushd twice wont work
better to make all one liner maybe please